ArduPlane/events.cpp

```diff
@@ -61,7 +61,7 @@ void Plane::failsafe_short_on_event(enum failsafe_state fstype, mode_reason_t re
 void Plane::failsafe_long_on_event(enum failsafe_state fstype, mode_reason_t reason)
 {
     // This is how to handle a long loss of control signal failsafe.
-    gcs().send_text(MAV_SEVERITY_WARNING, "Failsafe. Long event on, ");
+    gcs().send_text(MAV_SEVERITY_WARNING, "Failsafe. Long event on: type=%u/reason=%u", fstype, reason);
```

Comment:
This line is partially conveying the same information as events.cpp:118, one should be removed for the other.

Reply:
Removed the other one.